doc/development/deprecations.rst

```diff
@@ -9,6 +9,11 @@ deprecations are listed below.
 Pending deprecations
 --------------------
 
+* ``qml.transforms.map_batch_transform`` is deprecated, since transforms can be applied directly to a batch of tapes.
+  See :func:`~.pennylane.transform` for more information.
+
+  - Deprecated in v0.37
```

Comment:
We can put a target removal release of `v0.38`, since I don't think this is a particularly widely used function.

Reply:
Thanks @trbromley , I'll change this in the remaining open deprecation